python/pyspark/sql/column.py

```diff
@@ -185,17 +185,51 @@ def __contains__(self, item):
                          "in a string column or 'array_contains' function for an array column.")
 
     # bitwise operators
-    bitwiseOR = _bin_op("bitwiseOR")
-    bitwiseAND = _bin_op("bitwiseAND")
-    bitwiseXOR = _bin_op("bitwiseXOR")
+    _bitwiseOR_doc = """
+    Compute bitwise OR of this expression with another expression.
+
+    :param other: a value or :class:`Column` to calculate bitwise or(|) against
+                  this :class:`Column`.
+
+    >>> from pyspark.sql import Row
+    >>> df3 = spark.createDataFrame([Row(a=170, b=75)])
```

Comment:
Let me correct this as probably I guess you prefer `df` and I don't have preference.